app/formats.js

```diff
@@ -0,0 +1,30 @@
+export default {
```

Comment:
these will be used later if we support more locales or want to display formats a different way for some reason. It's more so we know this is a possibility with `ember-intl`

Reply:
The only problem I see is  with the `date` formats. Since `ember-intl` uses JS's `Intl.DateTimeFormat` to format time/datetime strings, a format with undefined options will result in that field not being included in the string. In this case, `{{format-date this.currentDateAndTime format='hhmmss'}}` will return `09:48 AM`. I am not sure whether including a `hhmmss` options for the date formats a good idea as it could cause confusion.

Reply:
Currently, we use `moment` to format times and datetimes.  A follow-up ticket will consist of internationalizing times and datetimes by replacing `moment` usage with `ember-intl.`